backend/src/models/user.ts

```diff
@@ -0,0 +1,35 @@
+import { Schema, model } from "mongoose";
+import { ProfileInterface } from "../shared/interface/modelInterface";
+
+const ProfileSchema: Schema<ProfileInterface> = new Schema({
+    "username": {
+        type: String,
+        required: true
+    },
+    "description": {
+        type: String,
+        default: ""
+    },
+    "dateOfBirth": {
+        type: String,
+        required: true
+    },
+    "year": {
+        type: String,
+        required: true
+    },
+    "major": {
+        type: String,
+        required: true
+    },
+    "musicPreference": {
+        type: String,
+        required: true
+    },
+    "favArtists": {
+        type: String,
+        required: true
+    },
+}, { versionKey: false });
```

Comment:
I think we should add field 'friend' and 'friend_requested' as well with type string[] storing id in a list